include/Network/HTTP/HTTPException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
```

Comment:
STANDARDS: please insert a line with ` * @file include/Network/HTTP/HTTPException.php`

It would be nice if you could add a doxygen header to the other exception files with at least the `@file` info. If you want to add a short `@brief` description this would be even better but I wouldn't expect it as part of this PR